tests/demo/test_demo.py

```diff
@@ -25,6 +25,7 @@ async def test_demo_repudiable_authentication_encryption(
         message=message,
         frm=ALICE_DID,
         to=BOB_DID,
+        pack_config=PackEncryptedConfig(forward=False)
```

Comment:
Please keep default values (so Forward=True) for demo tests.
Reason: this is not actually the tests, but rather examples how API should be used, what common params should be set (or not set). We expect to use forward=True by default, so it should not be changed in the demo scripts.

We may have a special version of DID DOCs for that demo tests that don't have routing keys, so that the message can be unpacked by Bob below.

Reply:
updated. please note that I set `forward=True` only for demo tests

makes sense to add though that many times these cases were very similar in relation to forwarding

Reply:
Do we really need to explicitly set Forward to True in the examples? Do we expect  a common user to care, or use a default (forward=True) in 95% of the cases?
I would prefer not to set it explicitly in most demo tests, as they try to show the minimal syntax (not be overloaded with details). We must set it explicitly in `demo_advanced_params` as the goal of the test to show all optional params.

Reply:
updated